src/components/SidebarItems.tsx

```diff
@@ -48,15 +48,33 @@ const SidebarItems: FC<{ cat: union[number]; active: boolean }> = ({
   const router = useRouter();
   const route = routeMap[cat];
 
+  const [mouseOver, setMouseOver] = useState(0);
+  const handleMouseOver = () => {
+    setMouseOver(1);
+    console.log(cat);
+  };
+
+  const handleMouseOut = () => {
+    setMouseOver(0);
+  };
+
   // This should never happen
   // Just so  TS doesn't complain
   if (!route) return null;
 
   return (
     <div
       className={`${
-        active ? ' -my-2.5 rounded-3xl bg-white py-2.5 shadow-md' : 'my-5'
+        (active || mouseOver) && cat != 'Events'
+          ? '-my-2.5 mb-2.5 rounded-3xl bg-white py-2.5 shadow-md'
+          : (active || mouseOver) && cat == 'Events'
+          ? '-my-2.5 rounded-3xl bg-white py-2.5 shadow-md'
+          : cat != 'Events'
+          ? 'mb-5'
+          : ''
```

Comment:
This ternary seems very complicated, maybe try using tailwind modifiers to see if there's a way you can reduce it into just one ternary instead of nested.

Reply:
you could also look into changing the whole thing into tailwind modifiers, but i don't know if that'd necessarily make the code better but might be an interesting learning experience/experiment.

Reply:
I've messed around with it a bit and I think switching it all to tailwind modifiers makes it more complicated. Not sure if thats just because I don't have a lot of experience with this stuff though.